sleuth_crawler/scraper/scraper/spiders/parsers/utils.py

```diff
@@ -28,4 +28,4 @@ def extract_element(item_list, index):
     try:
         return item_list[index].extract()
     except IndexError:
-        return ""
\ No newline at end of file
+        return ""
```

Comment:
I think it would be cleaner here to just check the `item_list` length before trying to access it at a particular index, thereby avoiding a try-catch block all together.

Reply:
fixed in b865341